frontend/src/components/basic/Listbox.tsx

```diff
@@ -27,86 +22,80 @@ interface ListBoxProps {
  * @param {function} obj.buttonAction - if there is a button at the bottom of the list, this is the action that happens when you click the button
  * @returns
  */
-export default function ListBox({
+const ListBox = ({
   selected,
   onChange,
   data,
   text,
   buttonAction,
-  isFull,
-}: ListBoxProps): JSX.Element {
+  isFull
+}: ListBoxProps): JSX.Element => {
   return (
     <Listbox value={selected} onChange={onChange}>
-      <div className='relative'>
+      <div className="relative">
         <Listbox.Button
           className={`text-gray-400 relative ${
             isFull ? 'w-full' : 'w-52'
           } cursor-default rounded-md bg-white/[0.07] hover:bg-white/[0.11] duration-200 py-2.5 pl-3 pr-10 text-left shadow-md focus:outline-none focus-visible:border-indigo-500 focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-orange-300 sm:text-sm`}
         >
-          <div className='flex flex-row'>
+          <div className="flex flex-row">
             {text}
-            <span className='ml-1 cursor-pointer block truncate font-semibold text-gray-300 capitalize'>
+            <span className="ml-1 cursor-pointer block truncate font-semibold text-gray-300 capitalize">
               {' '}
               {selected}
             </span>
           </div>
           {data && (
-            <div className='cursor-pointer pointer-events-none absolute inset-y-0 right-0 flex items-center pr-2'>
-              <FontAwesomeIcon icon={faAngleDown} className='text-md mr-1.5' />
+            <div className="cursor-pointer pointer-events-none absolute inset-y-0 right-0 flex items-center pr-2">
+              <FontAwesomeIcon icon={faAngleDown} className="text-md mr-1.5" />
             </div>
           )}
         </Listbox.Button>
         {data && (
           <Transition
             as={Fragment}
-            leave='transition ease-in duration-100'
-            leaveFrom='opacity-100'
-            leaveTo='opacity-0'
+            leave="transition ease-in duration-100"
+            leaveFrom="opacity-100"
+            leaveTo="opacity-0"
           >
-            <Listbox.Options className='border border-mineshaft-700 z-50 p-2 absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-bunker text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm'>
+            <Listbox.Options className="border border-mineshaft-700 z-50 p-2 absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-bunker text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
               {data.map((person, personIdx) => (
                 <Listbox.Option
-                  key={personIdx}
-                  className={({ active, selected }) =>
+                  key={`${person}.${personIdx + 1}`}
+                  className={({ active, selected: isSelected }) =>
```

Comment:
Looks really good! I think this thing should be 'selected'. This has to do with how Headless UI is made I guess. Otherwise, this leads to all the options being visually selected. I will try to fix this now. 

<img width="222" alt="Screenshot 2023-01-17 at 18 08 45" src="https://user-images.githubusercontent.com/78047717/213064326-36824a9a-f44b-4700-9500-f0516ae75205.png">